static/src/javascripts/projects/facia/modules/ui/container-show-more.js

```diff
@@ -12,28 +14,39 @@ define([
     qwery,
     _,
     $,
+    ajax,
+    config,
     mediator,
     userPrefs
 ) {
     var className = 'fc-show-more--hidden',
         textHook = 'js-button-text',
-        prefName = 'section-states';
+        prefName = 'section-states',
+        buttonSpinnerClass = 'collection__show-more--loading',
+        ARTICLE_ID_ATTRIBUTE = 'data-id',
+        ITEM_SELECTOR = '.js-fc-item',
+        STATE_DISPLAYED = 'displayed',
+        STATE_HIDDEN = 'hidden',
+        STATE_LOADING = 'loading';
```

Comment:
Why the mixture of constants in upper and lower case here? Just stick with one pattern.